markdown/about.md

```diff
@@ -9,9 +9,9 @@ The core team members who administer the nf-core project are listed below:
 * [@olgabot](https://github.com/olgabot): Olga Botvinnik
 * [@sven1103](https://github.com/sven1103): Sven F.
 
-We regularly review members - if you'd like to be involved, drop us a line on Slack.
+We regularly review members - if you'd like to be involved, drop us a message on Slack.
 
-All nf-core community members are expected to adhere to our [code of conduct](/code_of_conduct).
+All nf-core community members are expected to adhere to our [code of conduct](../CODE_OF_CONDUCT.md).
```

Comment:
Nope, this change will break the link on the website.

Needs a `replacementPatterns` to go to https://nf-co.re/code_of_conduct

Reply:
You can try this yourself. https://nf-co.re/CODE_OF_CONDUCT.md gives a 404

Reply:
I tried this locally with the MAMP setup and the link works fine for me.

Reply:
Huh, interesting - yes it works for me locally too. But not on the main site..

Ok, after a bit of digging I figured it out. Because MAMP is running on Mac (I guess), it doesn't care about URL capitalisation. For example, https://nf-co.re/About gives a 404, but http://localhost:8888/About works fine.

Sort of by accident, http://localhost:8888/CODE_OF_CONDUCT resolves to `code_of_conduct.php` which renders the markdown file. With MAMP this works even though the URL is upper case. On the nf-co.re website it fails.

Anyway, I think this needs to stay as /code_of_conduct then everything should be fine 👍 

```suggestion
All nf-core community members are expected to adhere to our [code of conduct](/code_of_conduct).
```

Reply:
huh yeah, weird! We should keep this in mind for the future.